src/plot_api/plot_api.js

```diff
@@ -2124,6 +2126,7 @@ Plotly.update = function update(gd, traceUpdate, layoutUpdate, traces) {
             data: restyleSpecs.eventData,
             layout: relayoutSpecs.eventData
         });
+        gd.emit('plotly_plotmodified');
```

Comment:
why not just subscribe to the existing [`plotly_afterplot`](https://github.com/plotly/plotly.js/blob/95c56e840aecf0c0a8a99a71341895f6bacbe4ab/src/plot_api/plot_api.js#L327) ?